src/wwwroot/js/genpage/main.js

```diff
@@ -455,7 +455,7 @@ function shiftToNextImagePreview(next = true, expand = false) {
 
 window.addEventListener('keydown', function(kbevent) {
     let isFullView = imageFullView.isOpen();
-    let isCurImgFocused = document.activeElement && 
```

Comment:
what

Reply:
same as the other. Empty space at the end, after `&&`